guides/common/modules/proc_creating-an-image-for-amazon-ec2.adoc

```diff
@@ -14,7 +14,7 @@ You can create images for Amazon EC2 from within {Project}.
 * In the *Username* field, enter the username needed to SSH into the machine.
 * In the *Image ID* field, enter the image ID provided by Amazon or an operating system vendor.
 ifndef::satellite[]
-This ID can be found within Amazon AWS or on operating system specific pages like https://wiki.debian.org/Cloud/AmazonEC2Image/Buster[debian.org] or https://cloud-images.ubuntu.com/locator/ec2/[ubuntu.com].
+This ID can be found within Amazon AWS or on operating system specific pages like https://cloud.debian.org/images/cloud/[debian.org] or https://cloud-images.ubuntu.com/locator/ec2/[ubuntu.com].
```

Comment:
```suggestion
You can find the ID in Amazon AWS or on operating system specific pages such as https://cloud.debian.org/images/cloud/[debian.org] or https://cloud-images.ubuntu.com/locator/ec2/[ubuntu.com].
```
Please change passive voice to active. 
The second suggestion  ("like" to "such as") is purely subjective, not required.

Reply:
Applied both suggestions.